daprdocs/content/en/operations/resiliency/resiliency-overview.md

```diff
@@ -0,0 +1,168 @@
+
+Policies can then be applied to [targets]({{< ref "targets.md" >}}), which include:
+
+- [Apps]({{< ref "targets.md#apps" >}}) via service invocation
+- [Components]({{< ref "targets.md#components" >}})
+- [Actors]({{< ref "targets.md#actors" >}})
+
+Additionally, resiliency policies can be [scoped to specific apps]({{< ref "component-scopes.md#application-access-to-components-with-scopes" >}}).
+
+Below is the general structure of a resiliency policy:
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Resiliency
+metadata:
+  name: myresiliency
+scopes:
+  # optionally scope the policy to specific apps
+spec:
+  policies:
+    timeouts:
+      # timeout policy definitions
+
+    retries:
+      # retry policy definitions
+
+    circuitBreakers:
+      # circuit breaker policy definitions
+
+  targets:
+    apps:
+      # apps and their applied policies here
+
+    actors:
+      # actor types and their applied policies here
+
+    components:
+      # components and their applied policies here
+```
+
+### Complete Example Policy
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Resiliency
+metadata:
+  name: myresiliency
+# simialrly to Subscriptions and Configurations specs, scopes lists the Dapr App IDs that this
+# configuration applies to.
+scopes:
+  - app1
+  - app2
+spec:
+  # policies is where timeouts, retries and circuit breaker policies are defined. 
+  # each is given a name so they can be referred to from the targets section in the resiliency spec.
+  policies:
+    # timeouts are simple named durations.
+    timeouts:
+      general: 5s
+      important: 60s
+      largeResponse: 10s
+
+    # retries are named templates for retry configurations and are instantiated for life of the operation.
+    retries:
+      pubsubRetry:
+        policy: constant
+        duration: 5s
+        maxRetries: 10
+
+      retryForever:
+        policy: exponential
+        maxInterval: 15s
+        maxRetries: -1 # retry indefinitely
+
+      important:
+        policy: constant
+        duration: 5s
+        maxRetries: 30
+
+      someOperation:
+        policy: exponential
+        maxInterval: 15s
+
+      largeResponse:
+        policy: constant
+        duration: 5s
+        maxRetries: 3
+
+    # circuit breakers are automatically instantiated per component and app endpoint.
+    # circuit breakers maintain counters that can live as long as the Dapr sidecar.
+    circuitBreakers:
+      simpleCB:
+        maxRequests: 1
+        timeout: 30s 
+        trip: consecutiveFailures >= 5
+
+      pubsubCB:
+        maxRequests: 1
+        interval: 8s
+        timeout: 45s
+        trip: consecutiveFailures > 8
+
+  # targets are what named policies are applied to. Dapr supports 3 target types - apps, components and actors
+  targets:
+    apps:
+      appB:
+        timeout: general
+        retry: important
+        # circuit breakers for services are scoped per endpoint (e.g. hostname + port).
```

Comment:
this e.g. is wrong is seems.  And endpoint is a method name which an app can have many off. If a circuit breaker applies to host + port then it applies to all endpoints (method) on a service. Is this correct?